docs/RELEASE_NOTES.md

```diff
@@ -26,6 +26,17 @@
 
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 -->
+### 5.0.0 - ???
+
+[Commits](https://github.com/commercetools/commercetools-sync-java/compare/4.0.1...5.0.0) |
+[Javadoc](https://commercetools.github.io/commercetools-sync-java/v/5.0.0/) | 
+[Jar](https://bintray.com/commercetools/maven/commercetools-sync-java/5.0.0)
+
+- 🚧 **Breaking Changes** (1)
```

Comment:
Not just callbacks, you need to add, update actions and utils. i.e: https://github.com/commercetools/commercetools-sync-java/blob/master/docs/RELEASE_NOTES.md#200---sept-14-2020

Reply:
cc: @ahalberkamp

Reply:
fb3b6c3df